Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```
@@ -0,0 +1,99 @@
+use std::io;
+use chrono::{NaiveDate, TimeDelta, Utc};
+use std::str::Split;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+struct Date {
+    year: i32,
+    month: u32,
+    day: u32,
+}
+
+fn convert_years_to_days () {
+
+    println!("Converting years to days...");
+    println!();
+    println!("Please enter the number of years to convert:");
+    let years_as_string  = get_input().unwrap();
+
+    let years_as_u32 = convert_string_to_u32(years_as_string.clone());
+
+    let years_in_days = NUMBER_OF_DAYS_IN_A_YEAR * years_as_u32;
+
+    println!("{years} years, is equal to {years_in_days} days", years = years_as_string.trim());
+}
+
+fn convert_string_to_u32 (input: String) -> u32 {
+
+    let value = u32::from_str(&input.trim()).unwrap();
+
+    value
+
+}
+
+fn get_input () -> io::Result<String> {
+
+    let mut buffer = String::new();
+
+    let stdin = io::stdin();
+
+    let _ = stdin.read_line(&mut buffer)?;
```

silently ignores error